proto/stream_plan.proto

```diff
@@ -56,6 +56,7 @@ message MaterializeNode {
 // Local and global aggregator distinguish with each other in PlanNode definition.
 message SimpleAggNode {
   repeated expr.AggCall agg_calls = 1;
+  // Not used for global simple agg.
   repeated uint32 distribution_keys = 2;
```

Comment:
why not deleted?

Reply:
`SimpleAggNode` is also used for local agg. 🤣